pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -599,13 +599,10 @@ func TestStorageRequests(t *testing.T) {
 			wantStatus: framework.NewStatus(framework.Unschedulable, getErrReason(v1.ResourceEphemeralStorage)),
 		},
 		{
-			pod: newResourceInitPod(newResourcePod(framework.Resource{EphemeralStorage: 25}), framework.Resource{EphemeralStorage: 25}),
+			pod: newResourceInitPod(newResourcePod(framework.Resource{EphemeralStorage: 5})),
 			nodeInfo: framework.NewNodeInfo(
-				newResourcePod(framework.Resource{MilliCPU: 2, Memory: 2})),
-			name: "ephemeral local storage request is ignored due to disabled feature gate",
-			features: map[featuregate.Feature]bool{
```

Comment:
Given we don't need features any more, can we clean up it from the test struct in L578?

Reply:
updated it to remove feature gate part. Thanks!